### ISSUE TITLE
AMP pages have cause issues with VoiceOver screen reader commands

### ISSUE THREAD

**Issue by mraccess77** [ampproject/amphtml#36107]
Perform a Google search and get an AMP page and you will find that using one of the resulting pages with a screen reader is very difficult and quickly reading through a page with different common commands does not work as expected.   Using the "Request desktop website" feature in Safari is a workaround for every AMP page that comes up in Google search results - but using that option should not be necessary for people who are blind or visually impaired.

* Features like explore by touch don't work as expected with the VoiceOver screen reader.  For example, touch someone where on the screen and the wrong item becomes focused.   
* Swiping left and right does work from the last focused item, however.  scrolling by page with three fingers up or down and then swipe left and right does not work correctly.   After a three finger scroll focus seems to get lost and goes back to the top of the page such as the heading level 1 after you use a left/right swipe.

**Comment by kristoferbaxter** [comment]
Hi @caroqliu, can you take a look at this issue?
